### ISSUE TITLE
[accessibility]: make sure that all the tap targets are atleast 48x48. 

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#1423]
As suggested here we must make sure that all the tap targets are at least 48x48 for better accessibility as suggested here https://support.google.com/accessibility/android/answer/7101858?hl=en

**Comment by Siddhant-K-code** [comment]
Can You Pls Specify the example buttons which are not good as of now, 

because I can see there are only a few taps like the stargazers button has a small size.

**Comment by nisarhassan12** [comment]
> Can You Pls Specify the example buttons which are not good as of now,

Hi, @Siddhant-K-code 🙂  Thanks for looking into it ❤️. Sorry I can not specify all of the tap targets that are not `48x48` right now i.e for that I'd have to go through the whole site to do that. I have a plan that when I'll work on this I'll go through all the important tappable elements on the entire site and fix them as I find them. 

If you want feel free to open up a Pr with the ones that you find right away and I'd be happy to take over from wherever you want to leave it off.

**Comment by Siddhant-K-code** [comment]
I figured out 3-4 things that are not `48X48`. Will make a PR for that.

It would be good if you can just give out one example, it will give more clarity to me.

**Comment by nisarhassan12** [comment]
Thanks @Siddhant-K-code The first thing that I see should be addressed is the nav toggle i.e its dimensions are 48x24

![image](https://user-images.githubusercontent.com/46004116/148521074-ef4ad343-3ddb-4e03-b5f3-736bac22df22.png)

The solution for the above would be to have invisible padding around the icon so that it is 48px in height you can add 12px spacing both to the top and the bottom to achieve that. Does this make sense ?

**Comment by nisarhassan12** [comment]
Hi, @Siddhant-K-code looks like you didn't have the time to work on this so I'm taking over this. Let me know in case you are still interested and want to continue on this. Thanks
